### DISCUSSION TITLE
Is there a html output template named msmb_html_book in Quarto?

### DISCUSSION THREAD

**Discussion by Wattskemov** [quarto-dev/quarto-cli#5450] [Q&A]
I have transfered from R Markdown to Quarto. Almost all features are suited by themself. But I think a html template in R Markdown naed named msmb_html_book missing in Quarto.

Is there a possible to add msmb_html_book to the output format html of Quarto book?

**Comment by mcanouil** [comment]
I don't know what that template is, but feel free to create/port this template as a Quarto extension, _i.e._ don't expect Quarto team to port all the templates ever made for `rmarkdown`.

See <https://quarto.org/docs/extensions/> to learn how to create format extension and to see several examples.
Additionally, see <https://github.com/mcanouil/awesome-quarto#templates>.

**Comment by cderv** [comment]
For reference, related issue opened upstream: https://github.com/grimbough/msmbstyle/issues/29
